Add AIChE 2023 post (#14)
SECQUOIA/SECQUOIA.github.io: _posts/2023-11-18-AIChE.md

---
layout: post
title: 2023 AIChE Annual Meeting
description: 'Reflecting on a Successful AIChE2023!'
image: assets/images/posts/AIChE_2023.jpeg
---

> We are thrilled to share highlights from our dynamic participation at AIChE2023, held last week in Orlando. Our team's presence was strongly felt across multiple sessions, showcasing our commitment to innovation and collaboration in chemical engineering.

Reviewer: ```suggestion
We are thrilled to share highlights from our dynamic participation at AIChE2023 in Orlando this 2023. Our team's presence was felt across multiple sessions, showcasing our commitment to innovation and collaboration in chemical engineering.
```

> Zedong Peng's presentation on MindtPy in the software for PSE session was a testament to our cutting-edge advancements in process systems engineering. His insights into this software tool were well-received, sparking engaging discussions among peers.

Reviewer: ```suggestion
Zedong Peng's presentation on MindtPy in the software for PSE session was well-received, sparking engaging discussions among peers.
```

> Andres F. Cabeza, our award-winning researcher, delivered a compelling talk on Tributyl Citrate process design in the #LatinXinChemE technical session. His groundbreaking work not only highlighted the importance of sustainable process design but also underscored our dedication to diversity and inclusion in the field.

Reviewer: ```suggestion
Andres F. Cabeza, our award-winning researcher, delivered a Tributyl Citrate process design talk in the #LatinXinChemE technical session. His groundbreaking work highlighted the importance of sustainable process design and our dedication to diversity and inclusion.
```

> Pedro Maciel Xavier made a significant impact with his poster on the QUBO.jl Julia environment. His contribution to this innovative project was evident and drew considerable interest from attendees. Additionally, his participation as a panelist in the #LatinXChemE ☕ panel was nothing short of inspiring, fostering meaningful dialogue on crucial topics within our community.

Reviewer: ```suggestion
Pedro Maciel Xavier presented his poster on the QUBO.jl Julia environment. Additionally, his participation as a panelist in the #LatinXChemE ☕ panel was inspiring, fostering meaningful dialogue on crucial topics within our community.
```

> We extend our heartfelt thanks to all who joined us, participated in our sessions, and contributed to the vibrant discussions. Your support and engagement are what make events like AIChE2023 truly memorable. Let's keep pushing the boundaries of chemical engineering together!

Reviewer: ```suggestion
We thank all who joined us, participated in our sessions, and contributed to the discussions. Your engagement is what makes events like AIChE2023 genuinely memorable. Let's keep pushing the boundaries of chemical engineering together!
```
